src/web/views/main.py

```
@@ -0,0 +1,282 @@
+            _task_store().create_task(
+                task_id,
+                title,
+                username=username,
+                form=request.form.to_dict(flat=True),
+            )
+        except TaskAlreadyExistsError as exc:
+            existing = exc.task
+            return redirect(
+                url_for("main.task1", task_id=existing["id"], title=title, error="task-active")
+            )
+        except Exception as exc:  # noqa: BLE001
```

Ruff (0.14.1)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)